Apparetments/AppartmentsCommands.c

```diff
@@ -1,10 +1,14 @@
-#define _CRT_SECURE_NO_WARNINGS
-#include <stdio.h>
-#include <string.h>
-#include <stdlib.h>
 #include "AppartmentsCommands.h"
 
-int appartmentCode = 0;
+unsigned short int appartmentCode = 0;
+
+void printWelcome()
+{
+	printf("Please enter one of the following commands:\n");
+	printf("add - apt, find - apt, buy - apt, delete - apt or exit\n");

```

Comment:
I think there shouldn't be spaces in the commands ( "add-apt" instead of "add - apt")

Reply:
true, gonna fix it